### PR TITLE
docs: improve getting started guide

### PR DESCRIPTION
**Pull request by ericmarcos**
## Description

Our getting started guide was very pushy trying to get users to signup to Hubtype. I've changed it to be more focused on learning core concepts. I've also added code snippets in code blocks for easy copy/paste, tips and images.

Screenshot 2020-11-26 at 17 12 20

Screenshot 2020-11-26 at 17 12 53

### CODE REVIEW

**Review by vanbasten17 — APPROVED** [review]
Very nice, I like it!!! 💯  
two little things, to be consistent with the changes i've done: 
- based on the "blank" template by running --> based on the "blank" example by running
- to see the list of all templates available --> to see the list of all examples available

**Comment by ericmarcos** [comment]
> Very nice, I like it!!! 💯
> two little things, to be consistent with the changes i've done:
> 
> * based on the "blank" template by running --> based on the "blank" example by running
> * to see the list of all templates available --> to see the list of all examples available

Hmmm let's move forward and let me think about this. It sounds really strange to me having a "blank example"

**Comment by vanbasten17** [comment]
I have changed all `templates` references to `examples` here, as we agreed: https://github.com/hubtype/botonic/pull/1134/commits/78c9ebd92ae610e98a9b1c80d299d74f64b71c9c, should I revert this commit then?

**Comment by ericmarcos** [comment]
> I have changed all `templates` references to `examples` here, as we agreed: [78c9ebd](https://github.com/hubtype/botonic/commit/78c9ebd92ae610e98a9b1c80d299d74f64b71c9c), should I revert this commit then?

No, no, it's ok. It's just the "blank" one that is weird. Normally you would expect an example to show you something... A "blank example" is like saying "here I show you how to achieve NOTHING" xD

**Comment by vanbasten17** [comment]
🤣

File: docs/website/sidebars.json [hunk 1]
```diff
@@ -95,8 +95,5 @@
       "releases/v0-10-1",
       "releases/v0-10-0"
     ]
-  },
-  "": {
```
Comment (elozano98) [inline docs/website/sidebars.json:99]: Do not we have FAQ anymore?

**Review by elozano98 — APPROVED** [review]

File: docs/website/sidebars.json (hunk 1 above)
Comment (ericmarcos) [reply docs/website/sidebars.json:99]: The FAQs page doesn't have a sidebar, so this is not needed.

**Pull request merged**